Getting-Started.adoc

```diff
@@ -61,36 +61,40 @@ helm search riff -l
 
 == [[current]]Install the current riff release
 
-We have separated the Kafka installation from the riff chart. This requires an extra step of installing Kafka in order to use riff. You can use the single-node Kafka chart provided by riff or the three-node kafka/zookeeper service provided by Kubeapps.
+We provide a lightweight single node Kafka installation with the `projectriff/kafka` chart. 
+This works well for development purposes and it can be installed together with the riff chart by providing `--set kafka.create=true` when installing the riff chart.
 
-=== Install Kafka chart
+=== Create riff-system namespace
 
-We provide a lightweight single node Kafka installation in the `projectriff/kafka` chart. This works well for development purposes.
+All riff components willbe deployed into a `riff-system` namespace. 
+Any functions that are developed can be deployed into any other namespace including the `default` namespace.
 
-Create the `riff-system` namespace:
+To create the `riff-system` namespace run:
 
 [source, bash]
 ----
 kubectl create namespace riff-system
 ----
 
-Install the kafka/zookeeper service provided by riff using:
+[TIP]
```

Comment:
I think we should move this whole tip down to a separate section (not under `Create riff-system namespace`)
Most of our users won't run the multi-node kafka.

We should also avoid mixing wide code blocks with text in tips becuase they make the text scroll sideways. Use tips just for the text, and keep the code block outside of the tip.

![screen shot 2018-04-06 at 1 56 13 pm](https://user-images.githubusercontent.com/849592/38436586-0195e788-39a3-11e8-9f61-357b1fdd7773.png)